.sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+# Add data about the pull request
+sonar.pullrequest.key=379
+sonar.pullrequest.branch=api_v2
+sonar.pullrequest.base=master
```

Comment:
sonarcloud already know about Github info, so it's not needed to be saved 

in fact, using a `.sonar-project.properties` is not advised for SonarCloud while we using the Automatic Analysis https://sonarcloud.io/documentation/analysis/automatic-analysis/#presence-of-a-properties-file

Reply:
I'll remove .sonar-project.properties and only keep .sonarcloud.properties

Reply:
success!! 
now there's only 1 issue being reported from the new work you've done: https://sonarcloud.io/project/issues?id=TheIOFoundation_ProjectLockdown&open=AXmAEfllz5W-InxO1660&pullRequest=379&resolved=false&types=CODE_SMELL
you can click `Why is this an issue?` to understand a bit more of, well, why ;)